sdk/template/template/test/public/configurationClient.spec.ts

```diff
@@ -6,8 +6,10 @@ import { Context } from "mocha";
 
 import { ConfigurationClient } from "../../src";
 
+import { isPlaybackMode, env } from "@azure-tools/test-recorder";
+import { NoOpCredential, TestProxyHttpClientCoreV1 } from "@azure-tools/test-recorder-new";
```

Comment:
Curious about the relationship/interaction between these two packages. Is it right to expect that packages would ordinarily use both?

Reply:
My understanding is those helpers will be eventually migrated to the new recorder as the old one is phased out... @HarshaNalluru, can you elaborate?

Reply:
Eventually, we'll remove recorder-new and put everything in `@azure-tools/test-recorder`. (mainly because `@azure-tools/test-recorder` has been already published).

So, the users will only need to import `@azure-tools/test-recorder`.

Reply:
> I'll consider moving the code to the @azure-tools/test-recorder package after this PR is merged and we locked the design.

That sounds great!